lib/Util/AutoPagingIterator.php

```diff
@@ -7,6 +7,7 @@ class AutoPagingIterator implements \Iterator
     private $lastId = null;
     private $page = null;
     private $params = array();
+    private $pageOffset = 0;
```

Comment:
Could you put this one line above so that the properties are in alphabetical order?